docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
The makefiles for Unix-like / Windows environments were auto-generated by Sphinx. I kept them as they are a nice convenience around the `sphinx-build` command, and most of their online documentation shows commands in the form `make ...`.